server/src/test/java/keywhiz/service/daos/SecretSeriesDAOTest.java

```diff
@@ -47,6 +50,51 @@
   @Inject @Readwrite SecretContentDAO secretContentDAO;
   @Inject @Readwrite GroupDAO groupDAO;
 
+  @Test public void renameUpdatesRowHmac() {
+    long id = createRandomSecretSeries();
+    String originalHmac = getRowHmac(id);
+
+    String newName = randomName();
+    secretSeriesDAO.renameSecretSeriesById(id, newName, "creator", ApiDate.now().toEpochSecond());
+    String renamedHmac = getRowHmac(id);
+
+    assertNotEquals(originalHmac, renamedHmac);
+  }
+
+  @Test public void createSecretSeriesUsesSuppliedId() {
+    long wellKnownId = 123;
+
+    long id = secretSeriesDAO.createSecretSeries(
+        wellKnownId,
+        "abc",
+        null,
+        null,
+        null,
+        null,
+        null,
+        ApiDate.now().toEpochSecond());
+
+    assertEquals(wellKnownId, id);
+  }
```

Comment:
This test is probably silly, it's on the same level as a getter test. Maybe I'm just being paranoid.